csfieldguide/static/interactives/flowcharts-interpreter-compiler/js/flowcharts-interpreter-compiler.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * JS for setting up the SVG arrows
+ * loosely based on:
+ * https://stackoverflow.com/questions/6278152/drawing-a-connecting-line-between-two-elements
+ * and:
+ * https://stackoverflow.com/questions/20037122/draw-an-arrow-between-two-divs
+ */
+
+var compiler = {
```

Comment:
Can you just initialise these vars in the document.ready function where you actually set the values of the keys?